src/main/resources/okta/js/applications.js

```diff
@@ -0,0 +1,618 @@
+;(function( $ ) {
```

Comment:
License Header is missing. Why do we have ";" in front

Reply:
This is from production code 
https://github.com/wso2/carbon-apimgt/blob/v6.1.66/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/js/applications.js#L1

Reply:
License header is added